### PR TITLE
Added menu button 

### PR DESCRIPTION
**Pull request by frnega**
in this project added menu button 
added  the [JavaScript + HTML + CSS linter](https://github.com/microverseinc/linters-config/tree/master/html-css-js).

### CODE REVIEW

File: menu.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  document.getElementsByClassName('About')[0].classList.add('hidden');
+  document.getElementsByClassName('form')[0].classList.add('hidden');
+};
+
+document
+  .getElementsByClassName('cancel')[0]
+  .addEventListener('click', closeMenu);
+document
+  .getElementsByClassName('menu-button-selector')[0]
+  .addEventListener('click', openMenu);
```
Comment (sja-thedude) [inline menu.js]: - [ ] Good job so far, however Minimum mobile screen size is 375 px which should fit the screen size but as you can see below it doesn't follow the design layout as per Figma and you should have made the mobile menu responsive until it changes to desktop screen size.

https://user-images.githubusercontent.com/70269507/195045182-c4cefdfd-f41b-4789-b8b9-9dbd208974bd.mov

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @frnega, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ Correct gitflow
✔️ Professional Readme
✔️ Linters passing

## Required 
- [ ] Kindly update your linters.yml with the html-css-js linters.yml which is in .github/workflows. As this file is not updated that's the reason eslint is not showing up in the checks.
- [ ] Kindly update your readme file by adding your and your partner's name to it. Also, make sure to add the license as a different file and link to it as per the microverse readme template.
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: menu.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
```
Comment (topeogunleye) [inline menu.js:26]: - [ ] As suggested by the previous reviewer. Please ensure that your mobile menu responsive until it changes to desktop screen size. This means the Close Menu Button should remain at the top right corner when increasing the device width.




https://user-images.githubusercontent.com/52934128/195071954-dbd518ad-fe5f-4446-9a92-6730c7ffc7bb.mp4

File: mit.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+Copyright 2022, [Frehiywot Nega]
+Please delete this line and the next one
+APP TYPE can be a webpage/website, a web app, a software and so on
+Permission is hereby granted, free of charge, to any person obtaining a copy of this [APP TYPE] and associated documentation files, to deal in the [APP TYPE] without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the [APP TYPE], and to permit persons to whom the [APP TYPE] is furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all copies or substantial portions of the [APP TYPE].
```
Comment (topeogunleye) [inline mit.md:6]: - [ ] [OPTIONAL] I kindly suggest that you replace `[APP TYPE]` with Web App to make the naming more appropriate.

**Review by topeogunleye — CHANGES_REQUESTED** [review]
Hi @frnega, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
* Good commit messages. :heavy_check_mark:
* No linter errors. :heavy_check_mark:
* Followed Github flow :heavy_check_mark:
* Professional README file. :heavy_check_mark:
* PR has a good title and summary :heavy_check_mark:

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

You can also consider:
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by nicupop729 — COMMENTED** [review]
Hello @frnega & @ernestkorir 👋,

_This is an additional review as a result of the second opinion request._

Even though I understand your point of view (as you say, it was not specified in the project requirements that the mobile menu needs to be responsive for multiple mobile screen sizes), I strongly encourage you to implement it as the recruiters will be very thoughtful about the look of your portfolio.  

As this was the only reason that your project was not approved it will be approved as soon as possible.
Please ask for another review in your Dashboard and the next Code Reviewer will mark it as approve right away :)

Keep rocking 🚀!

**Review by Davidosky007 — APPROVED** [review]
Hi @frnega 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
* No linter errors 👍 
* Well written readme file 👍 
* Responsive mobile menu 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**